setup-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org",
+    "$id": "http://yio-remote.com/yiodock.json",
+    "type": "object",
+    "title": "YIO Dock Integration Schema",
+    "description": "Required data points to set up a YIO Dock integration.",
+    "default": {},
+    "additionalProperties": true,
+    "required": [
+        "ip"
+    ],
+    "properties": {
+        "ip": {
+            "$id": "#/properties/ip",
+            "type": "string",
+            "title": "The ip schema",
+            "description": "The IP address or hostname of your YIO Dock.",
+            "default": "",
+            "examples": [
+                "192.168.100.2", "YIO-Dock-30AEA4D73C34.local"
```

Comment:
If also host names are allowed then the field should probably be named hostname (or something similar) instead of ip. Unless it's already used everywhere and a pain to change.

Reply:
We would need to change it here:
https://github.com/YIO-Remote/integrations.library/blob/master/src/yio-interface/configinterface.h#L39

And the schema file in all integrations.

Reply:
It will probably break some configurations, but it would make the meaning clear one can also use hostnames. You decide ;-)